### ISSUE TITLE
Maven User Settings

### ISSUE THREAD

**Issue by marecabo** [Dhruvkumarshah/maven-dependency-explorer#3]
First, thank you for your work at this extension!

Do you think it could be possible, to execute the `mvn dependency:tree` command also with the user settings as argument?
https://github.com/Dhruvkumarshah/maven-dependency-explorer/blob/2c19bf980d64fbe6d194281d7509584e49f9fb6d/src/extension.ts#L86 

Currently there are two related settings for this in VS Code from other Java extensions:

- `"java.configuration.maven.userSettings": "/path/to/settings.xml"` and
- `"maven.executable.options": "--settings /path/to/settings.xml -Pmyprofile"`

I think, the latter could be a good fit for this extension.

**Comment by Tordek947** [comment]
+1 here. On my project I had to use specific maven profiles on the Workspace basis. So I configured **maven.executable.options** in my settings.json so that the maven plugin works. But, unfortunately, this property is not utilised by _maven-dependency-explorer_, which might be beneficial.

I really love the way it displays the dependency tree, but I agree, that it can be supercharged even more by taking into consideration **maven.executable.options**

Thank you!
